### DISCUSSION TITLE
Logging in on multiple devices without getting logged out

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#7759] [Help]
Is it possible to allow the same user to login from multiple devices without being logged out? I am unable to login into the same app from two different browsers too!

**Comment by SantoshSrinivas79** [comment]
@melohagan @shogunpurple any idea on how to handle this? Is this a known issue?

**Reply by shogunpurple** [reply]
See https://github.com/Budibase/budibase/issues/5737 for more context. This is something that we will solve for non developer accounts, but the limitation was added to prevent several builder users sharing the same account

**Reply by SantoshSrinivas79** [reply]
yep .... agree ... need this only for users on production apps (no access to builder)
